client/src/client_sync/v21/wallet.rs

```diff
@@ -9,28 +9,28 @@
 //!
 //! See or use the `define_jsonrpc_minreq_client!` macro to define a `Client`.
 
-/// Implements Bitcoin Core JSON-RPC API method `createwallet` with descriptors=true (descriptor wallet)
+/// Implements Bitcoin Core JSON-RPC API method `createwallet` with descriptors=true (descriptor wallet).
 #[macro_export]
 macro_rules! impl_client_v21__create_wallet_with_descriptors {
     () => {
         impl Client {
             pub fn create_wallet_with_descriptors(&self, wallet: &str) -> Result<CreateWallet> {
                 let args = [
                     wallet.into(),
-                    false.into(),            // disable_private_keys
-                    false.into(),            // blank
-                    serde_json::Value::Null, // passphrase
-                    false.into(),            // avoid_reuse
-                    true.into(),             // descriptors=true
-                    serde_json::Value::Null, // load_on_startup
+                    false.into(),            // disable_private_keys.
+                    false.into(),            // blank.
+                    serde_json::Value::Null, // passphrase.
+                    false.into(),            // avoid_reuse.
+                    true.into(),             // descriptors=true.
+                    serde_json::Value::Null, // load_on_startup.
```

Comment:
These are incorrect. We only want to do rustdocs, in code comments if its not a full sentence then it doesn't get a full stop.